swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
 class Item:
-    pass
+    def __init__(self, category=None, condition=None):
+        self.category = category if category is not None else ""
+        self.condition = condition if condition is not None else 0
```

Comment:
For these default parameters, strings and numbers are immutable, so we don't really need to go this route for setting the values. We could use `""` and `0` as the default values directly.

Doing it this way can have some value since in theory it could allow us to detect the case where we are asking this constructor to use a default value (other than simply leaving off the argument). The way you structured your child class constructors benefits from this, since you can sort of "defer" picking the default value and have it only applied in the parent class. An alternative approach would be to set the default to `0` for Item and all its child classes, but then if we decided to change the defaults all to 5, we'd have to change it everywhere. With the way you set things up, you would only need to make that change here in Item.